docs/how-to-install-okteto-cli.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Install Okteto CLI
+description: Follow these steps to install the Okteto CLI, a client-side tool that works in any Kubernetes cluster
+sidebar_label: Installation
+id: installation
+---
+
+Okteto provides a local development experience for Kubernetes applications. You code locally in your favorite IDE and Okteto synchronizes it automatically to your cluster. The Okteto CLI is open source, and the code is available at [GitHub](https://github.com/okteto/okteto). It is a client-side only tool that works in any Kubernetes cluster.
+
+Install the Okteto CLI following these steps:
+
+### MacOS / Linux
+
+```console
+curl https://get.okteto.com -sSfL | sh
+```
+
+You can also install via [brew](https://brew.sh/) by running:
+
+```console
+brew install okteto
+```
+
+### Windows
+
+Download [https://downloads.okteto.com/cli/okteto.exe](https://downloads.okteto.com/cli/okteto.exe) and add it to your `$PATH`.
+
+You can also install via [scoop](https://scoop.sh/) by running:
+
+```console
+scoop install okteto
+```
+
+### GitHub
+
+Alternatively, you can directly download the binary [from GitHub](https://github.com/okteto/okteto/releases).
+
+#### Which binary should I download?
+
+First of all you need to check your OS and architecture to download the correct binary. You can check your OS by running:
+
+```console
+uname
+```
+
+You will also need to know the architecture which can be found by executing:
+
+```console
+uname -m
+```
+
+#### How to install a binary
+
+##### Linux/Mac
+
+You must give permissions to the binary by executing the instruction:
+
+```console
+chmod u+x $OKTETO_BIN
+```
+
+Then you have to move the `$OKTETO_BIN` to your `$PATH` by executing.
+
+```console
+mv -f $OKTETO_BIN /usr/local/bin/okteto
+```
+
+### Installing latest okteto release candidate
+
+You can test the latest features by installing the latest release candidate. Please note that the changes in these versions do not necessarily work with the latest stable version of okteto
+
+#### Linux/Mac
+
+```console
+curl https://get.beta.okteto.com -sSfL | sh
```

Comment:
instead of having duplicated scripts, how about we just tell people to directly download the RC build from github?

Reply:
I would have the script. I have seen many people having issues installing the binaries manually

Reply:
@jLopezbarb could we use the url `beta.okteto.com`?

Reply:
Another option is to have a single script, and base on an envar, like `OKTETO_VERSION` we install that version instead of the latest available.

Reply:
@jLopezbarb by default, the script should install the latest official release

Reply:
If we want want people to use a script to install the beta cli, then let's have a separate, so it doesn't require env vars.

Reply:
> @jLopezbarb could we use the url `beta.okteto.com`?

I added that as an example. I don't have access, but I suppose it's just creating a bucket with the script and create the endpoint, so it's possible to use `beta.okteto.com`